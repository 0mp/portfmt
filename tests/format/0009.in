OPTIONS_DEFINE= B A
