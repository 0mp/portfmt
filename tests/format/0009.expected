OPTIONS_DEFINE=	A B
